### DISCUSSION TITLE
Possible to store the notes / settings somewhere else?

### DISCUSSION THREAD

**Discussion by AuraofMana** [vigoren/foundryvtt-simple-calendar#261] [Ideas]
I noticed that all the notes you make in the calendar are stored in settings, which makes it hard because sometimes you want to update some other module then propagate that change across all your worlds. Well, when you copy settings around, you overwrite the notes written in the calendar. It would be great if these notes can be stored somewhere else.

**Comment by vigoren** [comment]
Hey, I am currently working on a large update for simple calendar. Part of this update is moving the storage of notes from being stored in the settings to being stored as journal entries.

So yes this will be addressed when the next update is released 😁
